cl/audio/tests.py

```diff
@@ -1,12 +1,43 @@
+from datetime import datetime
+from unittest import mock
+
 from django.urls import reverse
 from lxml import etree
 
+from cl.audio.factories import AudioWithParentsFactory
 from cl.audio.models import Audio
 from cl.lib.test_helpers import IndexedSolrTestCase, SitemapTest
 from cl.search.models import SEARCH_TYPES
+from cl.tests.fixtures import ONE_SECOND_MP3_BYTES, SMALL_WAV_BYTES
 
 
 class PodcastTest(IndexedSolrTestCase):
+
+    fixtures = [
+        "test_court.json",
+        "judge_judy.json",
+        "test_objects_search.json",
+        "authtest_data.json",
+    ]
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        cls.audio = AudioWithParentsFactory.create(
+            id=1,
+            source="C",
```

Comment:
Always prefer something like:

```suggestion
            source=SOURCES.COURT,
```

So we don't hard-code the letters here and there. Makes refactoring easier later.

Reply:
I think this wouldn't work exactly but Ill make an equivalent change